users/userModel.spec.js

```diff
@@ -12,15 +12,26 @@ describe('Model for users', () => {
   afterEach(() => db.migrate.rollback());
 });
 
-
 describe('/POST auth.register test suite', () => {
+  const user = {
+    firstname: 'John',
+    lastname: 'Doe',
+    email: 'johndoe@me.com',
+    password: '12345678',
+  };
+  it('should not have null values', () => {
```

Comment:
Good Job Anu. However i think a better approach should be to test what you expect to happen if it has or does not have a value.
for example. you have tested it('should not have null values'). so what happens when it has null values? 
`it(should fail if it has null values)`